.env

```diff
@@ -1,11 +1,12 @@
+NGINX_MLFLOW_STATIC_PORT=4242
+NGINX_PMML_STATIC_PORT=5353
+
 MODEL_1_PORT=1111
 MODEL_2_PORT=2222
 MODEL_3_PORT=3333
 MLFLOW_SERVER_PORT=5000
 POSTGRES_PORT=5432
 NGINX_BUCKET_PORT=9000
 
-NGINX_STATIC_PORT=4242
-
 PMML_SAMPLES_PORT=5100
```

Comment:
it is not for PMML (as I see you changed the name to NGINX_PMML_STATIC_PORT) - it is serving extra parametrs for model that are not provided for nussknacker and which should be downloaded additionally - it should be in something like 'abstract' docker comopse which can be used for mlfloe as well as for pmml as it is implemented in prinz api (not in pmml nor in mlflow)